doc/manual/metaprogramming.rst

```diff
@@ -9,106 +9,120 @@
 The strongest legacy of Lisp in the Julia language is its metaprogramming
 support. Like Lisp, Julia represents its own code as a data structure of
 the language itself.
+Because all data types and code in Julia are represented by Julia data structures,
+powerful reflection capabilities are available to explore the internals of a
+program and its types just like any other data.
 Since code is represented by objects that can be created and manipulated
```

Comment:
Moved to the end. But, on this point, I think repetition is quite a virtue :hammer:.